rl/function_approx.py

```diff
@@ -64,6 +84,14 @@ def update(self, xy_vals_seq: Sequence[Tuple[X, float]]) -> Dynamic[X]:
 
         return replace(self, values_map=new_map)
 
+    def solve(
+        self,
+        xy_vals_seq: Sequence[Tuple[X, float]],
+        error_tolerance: Optional[float] = None
+    ) -> Dynamic[X]:
+        new_map: Mapping[X, float] = {x: y for x, y in xy_vals_seq}
+        return replace(self, values_map=new_map)
```

Comment:
Is there a reason to define `new_map` as a variable here? Seems you could write it cleanly in one line:

```
return replace(self, values_map={x: y for x, y in xy_vals_seq})
```

If you have a sequence of tuples, you can turn it into a dictionary with the `dict` function, giving us:

```
return replace(self, value_map=dict(xy_vals_seq))
```

Reply:
Nice and simple. Made the change.